content/docs-v1/quire-commands.md

```diff
@@ -15,46 +15,60 @@ Quire commands are typically run from Terminal on a Mac and PowerShell on a PC.
+| **`quire pdf`** | Output Quire project in PDF format | You must run `quire build` before running this command. |
+| **`quire epub`** | Output Quire project in EPUB format | You must run `quire build` before running this command. |
+| **`quire clean`** | Remove old build outputs | |
 
 ## Get Help
 
-`quire -V` `quire --version`
-: Output the version number.
-
-`quire -h` `quire --help`
-: Provide a list of helpful Quire commands.
-
-`quire debug`
-: Development use only. Log info about current project.
+| Command       | Action   | Notes              |
+| ------------- | -------- | ------------------ |
+| **`quire --version`** | List Quire version number | This is useful for troubleshooting and confirming updates. |
+| **`quire --help`** | Provide list of Quire commands | This is useful if you need a reminder of commands. |
 
-## Helpful Non-Quire commands
 
-There are some additional commands that you will use that are not specific to Quire.
+## Coming Soon
 
-`cd`
-: Used to change project directories. You will use this command to navigate into your Quire project directory. Once you are in the directory you can run commands like `quire preview`, `quire install`, and `quire build`.
+These commands are not yet available but are currently under development.
 
-`cd ..`
-: Will take you back one directory level. This is especially useful if you keep multiple Quire projects as sub-directories within a main directory.
+| Command       | Action   | Notes              |
+| ------------- | -------- | ------------------ |
+| **`quire debug`** | Provide log into about current project | This will produce version numbers for the following: Quire, 11ty, Node, NPM, and your computer's operating system. This information will be requested when logging bugs and requesting troubleshooting support. |
+| **`quire configure`** | Edit the Quire CLI configuration | |
+| ** `quire preview pdf`** | Preview your Quire publication as a PDF |
+| **`quire preview epub`** | Preview your Quire publication as an EPUB |
+| **`quire server`** | Start a local web server to serve a previously built Quire site | |
+| **`quire version`** | Set the Quire version you want to use when running commands | This will be useful when switching between old and new Quire projects. |
+| **`quire version install`** | Install a different version of Quire | This will be useful when updating Quire or switching between projects built with different versions. |
+| **`quire version list`** | List installed versions Quire | This command does not include legacy versions of Quire. |
+| **`quire version prune`** | Remove outdated versions of Quire | This command does not include legacy versions of Quire. |
+| **`quire version remove`** | Uninstall versions of Quire | You can specify which version of Quire you would like to remove. |
```

Comment:
I think we should remove these, as they're more in the weeds and probably the furthest out from being implemented (if I had to guess)